maat-court-data-api/src/main/java/gov/uk/courtdata/filter/AddLaaTransactionIdToMDC.java

```diff
@@ -0,0 +1,37 @@
+package gov.uk.courtdata.filter;
+
+import gov.uk.courtdata.enums.LoggingData;
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.annotation.WebFilter;
+import jakarta.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@WebFilter("/*")
+@Component
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class AddLaaTransactionIdToMDC implements Filter {
+
+  @Override
+  public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+      throws IOException, ServletException {
```

Comment:
We can reuse both the filters in modernised services aswell by moving it to crime-commons. Currently, we log the laaTransactionId in all the MAAT Orchestration controllers. 

cc: @venkat980 @Mhowell494

Reply:
That's a very good point @skasthuri36326
I'll create a new PR to add these changes into crime-commons, once merged, I'll remove the same changes from this PR.

Reply:
I have created a new PR for crime-commons for the above.
@skasthuri36326 @Mhowell494 @venkat980 can one of you please review the PR below? Once done I can remove the same from this MAAT Court Data API PR.

https://github.com/ministryofjustice/laa-crime-commons/pull/105

Reply:
Approved. Created a minor release for crime-commons-classes. Could you please update the crime-commons-classes version to 3.20.0

Reply:
Have moved the functionality to `crime-commons` and verified that it is working as expected in maat court data api (see attached in PR description).
Using crime commons '3.21.0'